recipe/migrations/cuda130.yaml

```diff
@@ -0,0 +1,57 @@
+migrator_ts: 1755016036
+__migrator:
+  operation: key_add
+  migration_number:
+    1
+  build_number:
+    0
+  paused: false
+  override_cbc_keys:
+    - cuda_compiler_stub
+  check_solvable: false
+  primary_key: cuda_compiler_version
+  ordering:
+    cuda_compiler_version:
+      - 12.4
+      - 12.6
+      - 12.8
+      - None
+      - 12.9
+      - 13.0
+      # to allow manual opt-in for CUDA 11.8, see
+      # https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/7472
+      # must be last due to how cuda_compiler ordering in that migrator works
+      - 11.8
+  wait_for_migrators:
+    - cuda129
+  commit_message: |
+    Upgrade to CUDA 13.0
+
+    CUDA 13.0 removed support for architectures before `sm_75`, and renamed `sm_101` to
```

Comment:
Would be good to be explicit whether `sm_75` is dropped or still supported. Otherwise, people will likely interpret "before" inconsistently, as either `<`  or `≤`.

Reply:
e.g.
```suggestion
    CUDA 13.0 removed support for architectures `sm_72` and below, and renamed `sm_101` to
```

Reply:
I think sm_75 is Turing, and is still supported. I think the edit here makes good sense.

Reply:
Chose Bradley's wording in 521045c8abe8588cf3ff7952f9946a2d79a16c2d